jenkins/Jenkinsfile.linux

```diff
@@ -141,6 +93,7 @@ pipeline {
                 RSTUDIO_VERSION_SUFFIX) = utils.getVersion(!params.DAILY)
               RSTUDIO_VERSION_FLOWER = utils.getFlower()
               IS_PRO = RSTUDIO_VERSION_SUFFIX.contains('pro')
+              RSTUDIO_VERSION_FILENAME = utils.getVersionFilename(RSTUDIO_VERSION) // Define here for use later in utils.rebuildCheck()
```

Comment:
🤔

Reply:
As per our offline slack discussion, I'm going to ignore this, haha.

The reason this is done is because when `rebuildCheck()` runs in a `when` block, it has limited functionality and can't make these calls. Pre-defining it here allows `rebuildCheck()` to use this value later on.